cache/cache_test.go

```diff
@@ -0,0 +1,273 @@
+package cache
+
+import (
+	"context"
+	"fmt"
+	"testing"
+	"time"
+
+	"go.etcd.io/etcd/client/pkg/v3/types"
+	clientv3 "go.etcd.io/etcd/client/v3"
+	embed "go.etcd.io/etcd/server/v3/embed"
+)
+
+// two watchers see their own prefixes
+func TestDemuxIsolation(t *testing.T) {
```

Comment:
```suggestion
func TestCacheTwoWatchers(t *testing.T) {
```
Don't comment what can be expressed in name.